spec/ysh-string.test.sh

```diff
@@ -540,3 +540,101 @@ double
 ## status: 2
 ## stdout-json: ""
 
+#### str.find()
+
+var mystr = 'abcaz'
+
+echo $[mystr.find('b')]
```

Comment:
It would be easier to read to use the assert builtin in YSH now

e.g.

assert [1 === mystr.find('b')]
...

And then assert status: 0